### PR TITLE
sn-client and sn-redux update with missing actions, odata params, etc

### PR DESCRIPTION
**Pull request by herflis**
As the new documentation is growing, some of the examples point to missing stuff in the API. This pr adds the following actions, params and related tests:

sn-client:
- `enableautofilters` OData param
- `enablelifespanfilter` OData param

sn-redux
- `getChildrenCount` action to use ${path}/$count
- `getProperty` action to use ${path}/${fieldname}
- `getPropertyValue` action to use ${path}/${fieldname}/$value
- `getMetadata` action to user ${path}/$metadata

### CODE REVIEW

File: packages/sn-redux/src/Actions.ts [hunk 1]
```diff
@@ -519,3 +520,54 @@ export const setDefaultOdataOptions = (options: ODataParams<GenericContent>) =>
...
+export const getProperty = (idOrPath: string | number, propertyName: string) => {
+  const path = PathHelper.getContentUrl(idOrPath)
+  return {
+    type: 'GET_PROPERTY',
+    payload: (repository: Repository) => repository.fetch(`${path}/${propertyName}`),
+  }
+}
+
+/**
+ * Action creator for requesting a content's from sensenet Content Repository to get its value.
```
Comment (B3zo0) [inline packages/sn-redux/src/Actions.ts]:
```suggestion
 * Action creator for requesting a content's property from sensenet Content Repository to get its value.
```

File: packages/sn-redux/src/Actions.ts [hunk 2]
```diff
@@ -122,6 +122,7 @@ import {
 } from '@sensenet/client-core'
 import { GenericContent, User } from '@sensenet/default-content-types'
 import { PromiseMiddlewareAction } from '@sensenet/redux-promise-middleware'
+import { PathHelper } from '@sensenet/client-utils'
```
Comment (B3zo0) [inline packages/sn-redux/src/Actions.ts]: Please add client-utils to package.json and to tsconfig.json as well.

Comment (herflis) [reply packages/sn-redux/src/Actions.ts]: I've added it to the package.json and I noticed that it was already there in the tsconfig's references array.  Is it okay there, or it should be added to one of the other lists there (e.g. includes)?

Comment (B3zo0) [reply packages/sn-redux/src/Actions.ts]: I can't see sn-client-utils inside the references. It should be there.

**Pull request merged**